release-notes/10.0/preview/preview5/dotnetmaui.md

```diff
@@ -3,17 +3,93 @@
 Here's a summary of what's new in .NET MAUI, .NET for Android, and .NET for iOS, Mac Catalyst, macOS, and tvOS in this preview release:
 
 - .NET MAUI
-  - [Feature](#feature)
+  - [XAML Global Namespaces](#xaml-global-namespaces)
+  - [XAML Implicit Namespaces](#xaml-global-namespaces)
+  - [Intercept web requests](#intercept-web-requests)
 - [.NET for Android](#net-for-android)
 - [.NET for iOS, Mac Catalyst, macOS, tvOS](#net-for-ios-mac-catalyst-macos-tvos)
 
 .NET MAUI updates in .NET 10:
 
 - [What's new in .NET MAUI in .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10) documentation.
 
-## Feature
+## XAML Global Namespaces
 
-Something about the feature
+In your projects you can now glob together XML-namespaces into a new global namespace `xmlns="http://schemas.microsoft.com/dotnet/maui/global"`, and use them without prefixes. The .NET MAUI namespace is included already.
+
+By convention the new project template creates a file `GlobalXmlns.cs`. 
+
+```csharp
+[assembly: XmlnsDefinition(
+    "http://schemas.microsoft.com/dotnet/maui/global",
+    "MyApp.Views")]
+[assembly: XmlnsDefinition(
+    "http://schemas.microsoft.com/dotnet/maui/global",
+    "MyApp.Controls")]
+[assembly: XmlnsDefinition(
+    "http://schemas.microsoft.com/dotnet/maui/global",
+    "MyApp.Converters")]
+```
+
+You can then use anything in those namespaces like you do .NET MAUI controls, without prefix.
+
+```xml
+<ContentPage 
+    xmlns="http://schemas.microsoft.com/dotnet/maui/global"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="MyApp.MainPage">
+    <TagView x:DataType="Tag" />
+</ContentPage>
+```
+
+## XAML Implicit Namespaces
+
+You can opt-in to this feature by adding the following to your project file.
+
+```xml
+<PropertyGroup>
+  <DefineConstants>$(DefineConstants);MauiAllowImplicitXmlnsDeclaration</DefineConstants>
+  <EnablePreviewFeatures>true</EnablePreviewFeatures>
+</PropertyGroup>
+```
+
+With this change you can also eliminate `xmlns` and `xmlns:x` from XAML files.
+
+```xml
+<ContentPage x:Class="MyApp.MainPage">
+    <TagView x:DataType="Tag" />
+</ContentPage>
+```
```

Comment:
explain a bit:
- the default xmlns is the global one
- the x: prefix is added by default
- all xmlns with a XmlnsPrefix are also accessible with their prefix, even if they are included in the global xmlns, in case you need to disambiguate a name (the maui: prefix points to the maui xmlns)
- as the x: xmlns can not be glob'd, you still have to use the prefix (x:Class, x:DataType), but don't have to declare it